docs/resources/idp_ldap.md

```diff
@@ -38,6 +39,7 @@ resource "zitadel_idp_ldap" "default" {
 
 ### Required
 
+- `auto_linking` (String) Enable if users should get prompted to link an existing ZITADEL user to an external account if the selected attribute matches, supported values: AUTO_LINKING_OPTION_UNSPECIFIED, AUTO_LINKING_OPTION_USERNAME, AUTO_LINKING_OPTION_EMAIL
```

Comment:
Why is it required? Can't we default to AUTO_LINKING_OPTION_UNSPECIFIED? Like this we don't break existing resource configs.

Reply:
It is defined as required as the other options are defined as required.
And the resource field has AUTO_LINKING_OPTION_UNSPECIFIED as default, so did I miss something here?

Reply:
however...

![image](https://github.com/user-attachments/assets/d3fbadd5-503e-4fe2-97f0-85fa5a58d62f)